### PR TITLE
feat: create dashboard component in UI

### PR DESCRIPTION
**Pull request by gca-axelor**
RM#70817

### CODE REVIEW

**Comment by gca-axelor** [comment]
<PieChartDashboard
          data={[
            {
              value: 5,
              color: '#FFDF00',
              label: 'Draft',
            },
            {
              value: 3,
              color: '#ffa500',
              label: 'Waiting',
            },
            {
              value: 1,
              color: '#008000',
              label: 'Validate',
            },
            {
              value: 1,
              color: '#ff0000',
              label: 'Refused',
            },
          ]}
          legend={true}
        />

**Comment by gca-axelor** [comment]
<LineChartDashboard
          datasets={[
            {
              data: [
                {
                  value: 25,
                  color: '#FFDF00',
                  label: 'Draft',
                },
                {
                  value: 3,
                  color: '#ffa500',
                  label: 'Waiting',
                },
                {
                  value: 1,
                  color: '#008000',
                  label: 'Validate',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
              ],
            },
            {
              data: [
                {
                  value: 15,
                  color: '#ff0000',
                  label: 'Draft',
                },
                {
                  value: 3,
                  color: '#ffa500',
                  label: 'Waiting',
                },
                {
                  value: 4,
                  color: '#008000',
                  label: 'Validate',
                },
                {
                  value: 20,
                  color: '#ff0000',
                  label: 'Refused',
                },
              ],
            },
          ]}
        />
        <LineChartDashboard
          datasets={[
            {
              data: [
                {
                  value: 25,
                  label: 'Draft',
                },
                {
                  value: 3,
                  label: 'Waiting',
                },
                {
                  value: 3,
                  label: 'Waiting',
                },
              ],
            },
            {
              data: [
                {
                  value: 15,
                },
                {
                  value: 5,
                },
                {
                  value: 5,
                },
              ],
            },
          ]}
        />

**Comment by gca-axelor** [comment]
` <PieChartDashboard
          datasets={[
            {
              data: [
                {
                  value: 25,
                  color: '#FFDF00',
                  label: 'Draft',
                },
                {
                  value: 3,
                  color: '#ffa500',
                  label: 'Waiting',
                },
                {
                  value: 1,
                  color: '#008000',
                  label: 'Validate',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
              ],
            },
          ]}
          legend={true}
        />
        <PieChartDashboard
          datasets={[
            {
              data: [
                {
                  value: 25,
                  color: '#FFDF00',
                  label: 'Draft',
                },
                {
                  value: 3,
                  color: '#ffa500',
                  label: 'Waiting',
                },
                {
                  value: 1,
                  color: '#008000',
                  label: 'Validate',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
              ],
            },
          ]}
          legend={false}
        />
        <PieChartDashboard
          datasets={[
            {
              data: [
                {
                  value: 25,
                  label: 'Draft',
                },
                {
                  value: 3,
                  label: 'Waiting',
                },
                {
                  value: 1,
                  label: 'Validate',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
              ],
            },
          ]}
          legend={true}
        />`

**Comment by gca-axelor** [comment]
`<BarCharDashboard
          datasets={[
            {
              data: [
                {
                  value: 25,
                  label: 'Draft',
                  color: '#ff0000',
                },
                {
                  value: 3,
                  label: 'Waiting',
                },
                {
                  value: 1,
                  label: 'Validate',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
              ],
            },
          ]}
        />
        <BarCharDashboard
          datasets={[
            {
              data: [
                {
                  value: 25,
                  label: 'Draft',
                },
                {
                  value: 3,
                  label: 'Waiting',
                },
                {
                  value: 1,
                  label: 'Validate',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
                {
                  value: 1,
                  color: '#ff0000',
                  label: 'Refused',
                },
              ],
            },
          ]}
          spacing={60}
        />`

File: packages/ui/src/components/molecules/LineChartDashboard/LineChartDashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  color?: string;
+  label: string;
+}
+
+interface Dataset {
+  data: Data[];
+}
+
+interface LineChartDashboardProps {
+  datasets: Dataset[];
```
Comment (vhu-axelor) [inline packages/ui/src/components/molecules/LineChartDashboard/LineChartDashboard.tsx:36]: datasets: Data[][];

File: packages/ui/src/components/molecules/PieChartDashboard/PieChartDashboard.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  legend: boolean;
+}
+
+const PieChartDashboard = ({datasets, legend}: PieChartProps) => {
+  const Color = useThemeColor();
+
+  const [dataSet, setDataSet] = useState(datasets[0]?.data);
+
+  useEffect(() => {
+    console.log(datasets[0]?.data);
```
Comment (vhu-axelor) [inline packages/ui/src/components/molecules/PieChartDashboard/PieChartDashboard.tsx:47]: Don't forget to remove it

File: packages/ui/src/components/molecules/index.tsx [hunk 3]
```diff
@@ -19,6 +19,7 @@
 export {default as Alert} from './Alert/Alert';
 export {default as AttachmentCard} from './AttachmentCard/AttachmentCard';
 export {default as Badge} from './Badge/Badge';
+export {default as BarCharDashboard} from './BarCharDashboard/BarCharDashboard';
```
Comment (vhu-axelor) [inline packages/ui/src/components/molecules/index.tsx:22]: BarChartDashboard (with a t)

**Review by vhu-axelor — COMMENTED** [review]
Nice dashboards 🤩️

**Pull request closed without merging**